make_new_tables.py

```diff
@@ -113,6 +116,10 @@ def main():
     datecols = ['datestart', 'datestart']
     tables = ['load_segments', 'timelines']
 
+    if opt.tl_processing:
+        tables.extend(['tl_obsids', 'tl_built_loads', 'tl_processing'])
+        datecols.extend([None, None, None])
```

Comment:
These do have date columns that could have been used for filtering, but I didn't see a driver.